app/api/v2/models/base_model.py

```diff
@@ -155,3 +155,42 @@ def validate_token_status(self, token):
         except jwt.InvalidTokenError:
 
             return "This token is invalid"
+
+    def check_user_is_admin(self, user_id):
+        """ Checks if a user is an admin using the user_id """
+        database = create_tables()
+
+        cur = database.cursor()
+        cur.execute(
+            """ SELECT user_id FROM users WHERE roles = 'true'; """)
```

Comment:
Why are you checking if column roles is `true` why not have roles as either `admin` or `user`